<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAnswerQuestion</class>
 <widget class="QDialog" name="DialogAnswerQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton {
	background-color: rgb(0, 157, 66);
	height: 113;
	width: 113;
	color: white;
	font: 74pt &quot;微软雅黑&quot;;
}

QDialog {
	border: 6px solid rgb(166, 174, 101);
	background-color:white;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="tbChoiceA">
     <property name="text">
      <string>A</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbChoiceB">
     <property name="text">
      <string>B</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbChoiceC">
     <property name="text">
      <string>C</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbChoiceD">
     <property name="text">
      <string>D</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
